cdk_framework/EKS/lib/config/cluster-config/clusters.yml

```diff
@@ -1,16 +1,9 @@
 ---
 clusters:
-  amdCluster02:
-    launch_type: 
-      fargate:
-    version: 1.21
-  fargateCluster01:
-    launch_type: 
-      fargate:
-    version: 1.21
-  t4gCluster01:
-    launch_type: 
-      ec2:
-        ec2_instance: m5
-        node_size: large
-    version: 1.21
+  - name: ec2Cluster
+    version: "1.21"
```

Comment:
Why does the `version` field require double quotes?

Reply:
So that it is of type string.  https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-eks-cluster.html#cfn-eks-cluster-version This is a link that Raphael sent me that seems to support this.